### PR TITLE
[12.0][MIG] website_snippet_country_dropdown

### PR DESCRIPTION
**Pull request by Tardo**
Some changes:
Odoo 12.0 uses Bootstrap4 and in this version 'btn-default' has been renamed to 'btn-secondary'... this change a bit the style of the button compared with 10.0

### CODE REVIEW

File: website_snippet_country_dropdown/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Alexandre Díaz
```
Comment (yajo) [inline website_snippet_country_dropdown/readme/CONTRIBUTORS.rst]: Don't forget the other contributors!

**Review by yajo — APPROVED** [review]
The CLA bot message seems irrelevant. Code review.

File: website_snippet_country_dropdown/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (Tardo) [reply website_snippet_country_dropdown/readme/CONTRIBUTORS.rst]: Done, sorry for the mistake :/

**Comment by yajo** [comment]
Thanks! Please keep migration changes in one commit :blush:

File: website_snippet_country_dropdown/README.rst [hunk 2]
```diff
@@ -26,41 +50,10 @@ The template have three input text:
 The name of the complete field is customizable when user insert the snippet
 into a form element with the website editor.
 
-Development
```
Comment (pedrobaeza) [inline website_snippet_country_dropdown/README.rst]: Don't remove all of this. You have to put it inside `USAGE.rst` file with a subtitle level like:
```rst
Development
~~~~~~~~~~~
```

File: website_snippet_country_dropdown/README.rst [hunk 3]
```diff
@@ -26,41 +50,10 @@ The template have three input text:
...
-
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/186/9.0
-
 Known issues / Roadmap
 ======================
 
 * Add tests.
-* Flag images should be lazy-loaded to optimize loading.
```
Comment (pedrobaeza) [inline website_snippet_country_dropdown/README.rst]: Is this already done?

File: website_snippet_country_dropdown/demo/assets.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
-     © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+<!-- (C) 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
```
Comment (pedrobaeza) [inline website_snippet_country_dropdown/demo/assets.xml]: Use `Copyright` full word instead of abbreviations

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Apart from comments, I have observed 2 things:

- The color of the dropdown is now purple, while in v10 it was white. Is it due to used variable have changed?
- The label appears in a different line than the dropdown.

File: website_snippet_country_dropdown/README.rst (hunk 3 above)
Comment (Tardo) [reply website_snippet_country_dropdown/README.rst]: Now yes, sorry for the mistake :/

File: website_snippet_country_dropdown/demo/assets.xml (hunk 4 above)
Comment (Tardo) [reply website_snippet_country_dropdown/demo/assets.xml]: Done

**Comment by Tardo** [comment]
@pedrobaeza The style changes a bit... but now looks better: uses proper bootstrap4

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**